examples/Watchdog_live_view_demo.ipynb

```diff
@@ -0,0 +1,489 @@
+      "application/vnd.holoviews_load.v0+json": ""
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import discretisedfield as df\n",
+    "import xarray as xr\n",
+    "import holoviews as hv\n",
+    "import numpy as np\n",
+    "import time\n",
+    "import os\n",
+    "import panel as pn\n",
+    "import json\n",
+    "\n",
+    "from holoviews.streams import Pipe\n",
+    "from watchdog.observers import Observer\n",
+    "from watchdog.events import FileSystemEventHandler\n",
+    "\n",
+    "hv.extension(\"bokeh\", logo=False)\n",
+    "pn.extension()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is the class that contains the functionality for the live visualisation. We overwrite methods from the watchdog abstract class FileSystemEventHander."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class DirectoryMonitorHandler(FileSystemEventHandler):\n",
+    "    def __init__(self, observer, stop_flag):\n",
```

Comment:
I think you need to pass `working_directory` to ensure you are looking for filesystem events in the right place.